test/truffle/ecosystem/launchers.sh

```diff
@@ -20,8 +20,8 @@ cd "$(jt ruby -e 'puts Truffle::Boot.ruby_home')"
 function check_launchers() {
     if [ -n "$2" ]
     then
-        [[ "$(${1}truffleruby --version)" =~ truffleruby\ .*\ like\ ruby\ 2.3.5 ]]
-        [[ "$(${1}ruby --version)" =~ truffleruby\ .*\ like\ ruby\ 2.3.5 ]]
+        [[ "$(${1}truffleruby --version)" =~ truffleruby\ .*\ like\ ruby\ 2.3.6 ]]
+        [[ "$(${1}ruby --version)" =~ truffleruby\ .*\ like\ ruby\ 2.3.6 ]]
```

Comment:
There is also `spec/truffle/launcher_spec.rb`

Reply:
Ah, missed that. I'd searched for `2.3.5` and `2_3_5`, but not the regexp escaped `2\.3\.5`.

Reply:
I've added notes to the docs about searching for the ruby version number, and other version numbers such as gem if they have changed.